pkg/defaults/policies/files/cvss_7.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "f09f8da1-6111-4ca0-8f49-294a76c65115",
-  "name": "Fixable CVSS >= 7",
-  "description": "Alert on deployments with fixable vulnerabilities with a CVSS of at least 7",
+  "name": "Containers with fixable CVEs and CVSS >= 7",
+  "description": "Alert on containers with fixable vulnerabilities with a CVSS of at least 7. While we suggest to rely on Severity instead of CVSS by default, you may want to favor CVSS, for example, because of FedRAMP requirements. If you decide to use this policy, consider disabling 'Containers with Fixable CVEs and Severity at least Important' to avoid multiple violations for the same problem.",
```

Comment:
alerts for policy violations are raised on deployment level. hence should be worded accurately - Deployment with containers with fixable CVEs and CVSS >= 7